io.github.everestapi.Olympus.yml

```diff
@@ -0,0 +1,61 @@
+app-id: io.github.everestapi.Olympus
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: olympus
+finish-args:
+  # Allow the GUI to render
+  - --socket=x11
+  - --share=ipc
+  # Needed to access mod list and download mods
+  - --share=network
+  # Allow OpenGL acceleration
+  - --device=dri
+  # Needed to get the sound working
+  - --socket=pulseaudio
+  # Needed to launch celeste from Olympus
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
This needs an exception in the linter.

How does Olympus work? Will it launch Celeste from steam and this is for that?

Reply:
Olympus is able to directly launch Celeste so in case of a steam install the game itself invokes steam. It needs `flatpak-spawn` to be able to launch the game system-wide thus not requiring any drivers or special libs inside the flatpak. `flatpak-spawn` is only used in order to start the game outside the sandbox

Reply:
PR to https://github.com/flathub/flatpak-builder-lint, eg. https://github.com/flathub/flatpak-builder-lint/blob/711f312a2f0e367447670e73d58b74ac290ff4fc/flatpak_builder_lint/staticfiles/exceptions.json#L1551-L1553

Reply:
Corresponding PR : https://github.com/flathub/flatpak-builder-lint/pull/191